CONTRIBUTING.md

```diff
@@ -1,82 +1,79 @@
 # Lawnchair Launcher Contributing Guidelines
 
-![](https://raw.githubusercontent.com/validcube/lawnchair/pave-path/.github/lawnchair_round.png)
+<img title="Lawnchair Launcher logo" alt="Lawnchair Launcher logo" src="https://raw.githubusercontent.com/validcube/lawnchair/pave-path/.github/lawnchair_round.png" width="100" data-align="inline">
```

Comment:
It's recommended to set alt text to none to avoid unnecessary reading on accessibility tool.

```suggestion
<img alt="" src="https://raw.githubusercontent.com/validcube/lawnchair/pave-path/.github/lawnchair_round.png" width="100" data-align="inline">
```